ocpp/charge_point.py

```diff
@@ -224,7 +224,7 @@ async def _handle_call(self, msg):
             # when no '_on_after' hook is installed.
             pass
 
-    async def call(self, payload):
+    async def call(self, payload, surpress=True):
```

Comment:
s/surpress/suppress

Reply:
Can you also update the docstring and mention this new argument?

Reply:
I would change the name to "suppress_ocpp_exception" or "suppress_exception". 

Or later on in the code I dont remember what the suppress is suppressing

Reply:
I would say that keeping the name simple would be preferable. I hope the docstring explains what the argument's purpose is.

Reply:
ok, fair enough